src/v2/Apps/Authentication/Components/ModalContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import Cookies from "cookies-js"
 import { Component } from "react"
-import { data as sd } from "sharify"
+import { data as sd } from "sharify" // eslint-disable-line no-restricted-imports
```

Comment:
It seems out of scope to refactor this component to stop using sharify.

Reply:
Mind adding a ticket in the web practice board [here](https://artsyproduct.atlassian.net/jira/software/projects/WP/boards/97) to remove this flag? Can future friday the fix.